### PR TITLE
Update typoCorrector to check for newly implemented commands

### PR DESCRIPTION
**Pull request by HUANGXUANKUN**
@WEIFENG-NUSCEG  It is to add in command keywords requested by weifeng for his newly implemented AssignCommand and DeleteCommand

### CODE REVIEW

File: src/main/java/duke/core/TypoCorrector.java [hunk 1]
```diff
@@ -20,13 +20,13 @@
...
+        Arrays.asList("bye", "duke", "help"));
     private static final ArrayList<String> twoKeywordsCommands = new ArrayList<String>(
-            Arrays.asList("list patients", "list tasks"));
+        Arrays.asList("list patients", "list tasks"));
     private static final ArrayList<String> otherCommands = new ArrayList<String>(
-            Arrays.asList("update patient", "update task",
-                    "delete patient", "delete task", "add task", "add patient",
-                    "assign by", "find patient", "find task"));
+        Arrays.asList("update patient", "update task",
+            "delete patient", "delete task", "delete patienttask", "add task", "add patient",
```
Comment (lmtaek) [inline src/main/java/duke/core/TypoCorrector.java]: Parser has been implemented in a way that we don't have to compensate for spacing with keywords such as `patienttask`; in this case, `delete patient task` should work fine.

File: src/main/java/duke/core/TypoCorrector.java [hunk 2]
```diff
@@ -20,13 +20,13 @@
...
+            "find patient", "find task", "assign standardtask", "assign eventtask"));
```
Comment (lmtaek) [inline src/main/java/duke/core/TypoCorrector.java]: Same brought up in my earlier comment; no need to squish together `standardtask`. `assign standard task` and `assign event task` should be expected to work.

File: src/main/java/duke/core/TypoCorrector.java (hunk 1 above)
Comment (HUANGXUANKUN) [reply src/main/java/duke/core/TypoCorrector.java]: Noted.

File: src/main/java/duke/core/TypoCorrector.java [hunk 3]
```diff
@@ -19,14 +16,10 @@
...
-    private static final ArrayList<String> twoKeywordsCommands = new ArrayList<String>(
-            Arrays.asList("list patients", "list tasks"));
-    private static final ArrayList<String> otherCommands = new ArrayList<String>(
-            Arrays.asList("update patient", "update task",
-                    "delete patient", "delete task", "add task", "add patient",
-                    "assign by", "find patient", "find task"));
+    private static final String[] simpleCommands = {"bye", "duke", "help", "list patients", "list tasks"};
+    private static final String[] otherCommands = {"update patient", "update task", "delete patient",
+        "delete task", "delete assigned task", "add task", "add patient", "find patient", "find task",
+        "find assigned task", "assign deadline task", "assign event task"};
```
Comment (lmtaek) [inline src/main/java/duke/core/TypoCorrector.java]: `find assigned task` should be `find assigned tasks`, but this is a small fix!

Comment (HUANGXUANKUN) [reply src/main/java/duke/core/TypoCorrector.java]: Noted! Will fix it in the next PR

**Pull request merged**